visualization/app/codeCharta/ui/detailPanel/detailPanel.html

```diff
@@ -57,8 +57,15 @@
             <tr>
                 <td><i class='fa fa-location-arrow'></i></td>
                 <td>Path</td>
-                <td ng-hide="!$ctrl.isSelected()">{{$ctrl.details.selected.path}}</td>
-                <td ng-hide="!$ctrl.isHovered()">{{$ctrl.details.hovered.path}}</td>
+                <td  ng-hide="!$ctrl.isSelected()">
+                    <a ng-if="$ctrl.details.selected.link" target="_blank" href="{{$ctrl.details.selected.link}}">{{$ctrl.details.selected.path}}</a>
+                    <a ng-if="!$ctrl.details.selected.link">{{$ctrl.details.selected.path}}</a>
+                </td>
+
+                <td  ng-hide="!$ctrl.isHovered()">
+                    <a ng-if="$ctrl.details.hovered.link" target="_blank" href="{{$ctrl.details.hovered.link}}">{{$ctrl.details.hovered.path}}</a>
```

Comment:
I am not sure if we should show the link for hovered buildings. On one hand it is visually symmetric on the other hand it should technically not be possible to click on the hovered elements link since you unhover it by moving your mouse.

Reply:
Yes, it usually is not possible to click on the hovered link, but you still can select the building or just double click. So you prefer not to show the hovered link? I would just keep it, because if I hover a building and see "oh there's no link", I might think there is even no link, when I select the building.

Reply:
In my opinion the link does not break anything and provides, like you mentioned, visual information for the user.